backend/packages/Upgrade/src/api/services/ExperimentUserService.ts

```diff
@@ -159,15 +161,18 @@ export class ExperimentUserService {
       const { originalUser, ...rest } = user;
       return { ...rest, originalUser: originalUser.id };
     });
+
+    let aliasesToReturn = alreadyLinkedAliases.map((alias) => alias.id);
+
     if (userAliasesDocs.length) {
       let aliasesUsers = await this.userRepository.save(userAliasesDocs);
       aliasesUsers = aliasesUsers.map((user) => {
         const { originalUser, ...rest } = user;
         return { ...rest, originalUser: originalUser.id };
       });
-      return [...aliasesUsers, ...alreadyLinkedAliases];
+      aliasesToReturn = [...aliasesToReturn, ...aliasesUsers.map((alias) => alias.id)];
     }
-    return alreadyLinkedAliases;
+    return { ...userToReturn, aliases: aliasesToReturn };
```

Comment:
Instead of `...userToReturn` just use `userId: ...`

Reply:
code updated